docs/solutions/k8s-on-dcos.md

```diff
@@ -0,0 +1,271 @@
+# Blueprint - Kubernetes on the Mesosphere DCOS
```

Comment:
Yup, we're talking about the same and yes, let's ask @jdef maybe he knows … once it's fixed I can update it here as well.